deployment_config/prod_vars.yml

```diff
@@ -6,3 +6,4 @@ REDIRECT_URI_HOST: https://ttahub.ohs.acf.hhs.gov
 rds_instance: ttahub-prod
 s3_doc_upload_bucket: ttahub-document-upload-prod
 TTA_SMART_HUB_URI: https://ttahub.ohs.acf.hhs.gov
+CLAMAV_ENDPOINT: http://clamapi-ttahub-dev.apps.internal:8080
```

Comment:
This one should be `http://clamapi-ttahub-prod.apps.internal:8080`

Reply:
good catch, I must have copied the wrong one from the user story.